<ui version="4.0" >
 <class>modCalcVlsrDlg</class>
 <widget class="QWidget" name="modCalcVlsrDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>548</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>458</width>
        <height>492</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Date and time:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QDateTimeEdit" name="Date" />
           </item>
           <item>
            <widget class="KPushButton" name="NowButton" >
             <property name="text" >
              <string>Now</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="KPushButton" name="LocationButton" >
           <property name="text" >
            <string>Greenwich, United Kingdom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>331</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>Target position:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" >
          <widget class="KPushButton" name="ObjectButton" >
           <property name="text" >
            <string>Select Object...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Right ascension:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="dmsBox" name="RA" >
           <property name="minimumSize" >
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Declination:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="dmsBox" name="Dec" />
         </item>
         <item row="2" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="4" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>321</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="4" >
          <widget class="QLabel" name="label_5" >
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>Radial velocities:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1" >
          <widget class="QLabel" name="azLabel" >
           <property name="text" >
            <string>V&lt;sub>LSR&lt;/sub>:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" >
          <widget class="KLineEdit" name="VLSR" />
         </item>
         <item row="5" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1" >
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Heliocentric:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2" >
          <widget class="KLineEdit" name="VHelio" />
         </item>
         <item row="6" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1" >
          <widget class="QLabel" name="textLabel2" >
           <property name="text" >
            <string>Geocentric:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" >
          <widget class="KLineEdit" name="VGeo" />
         </item>
         <item row="7" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="1" >
          <widget class="QLabel" name="textLabel3" >
           <property name="text" >
            <string>Topocentric:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2" >
          <widget class="KLineEdit" name="VTopo" />
         </item>
         <item row="8" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="3" column="0" >
             <widget class="QCheckBox" name="DecCheckBatch" >
              <property name="text" >
               <string>Dec:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="3" >
             <widget class="KLineEdit" name="EpochBoxBatch" />
            </item>
            <item row="2" column="2" >
             <widget class="QCheckBox" name="ElevationCheckBatch" >
              <property name="text" >
               <string>Elevation (m):</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="RACheckBatch" >
              <property name="text" >
               <string>RA:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3" >
             <widget class="dmsBox" name="LongitudeBoxBatch" />
            </item>
            <item row="3" column="2" >
             <widget class="QCheckBox" name="EpochCheckBatch" >
              <property name="text" >
               <string>Epoch:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QCheckBox" name="LongCheckBatch" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="3" >
             <widget class="KLineEdit" name="ElevationBoxBatch" />
            </item>
            <item row="1" column="2" >
             <widget class="QCheckBox" name="LatCheckBatch" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QTimeEdit" name="UTBoxBatch" />
            </item>
            <item row="2" column="1" >
             <widget class="dmsBox" name="RABoxBatch" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="dmsBox" name="DecBoxBatch" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="UTCheckBatch" >
              <property name="text" >
               <string>UT:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3" >
             <widget class="dmsBox" name="LatitudeBoxBatch" />
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="DateCheckBatch" >
              <property name="text" >
               <string>Date:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDateEdit" name="DateBoxBatch" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="InputVelocityCheckBatch" >
              <property name="text" >
               <string>Input velocity:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="InputVelocityComboBatch" >
              <item>
               <property name="text" >
                <string>Heliocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Geocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Topocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>LSR</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="KLineEdit" name="InputVelocityBoxBatch" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Select Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AllRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="OutputFileBoxBatch" />
          </item>
          <item row="0" column="1" >
           <widget class="KUrlRequester" name="InputFileBoxBatch" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Input file:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="RunButtonBatch" >
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="overview" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Overview</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="KTextBrowser" name="OverviewBox" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
